.github/workflows/ci.yml

```diff
@@ -54,7 +55,7 @@ jobs:
       fail-fast: false
       matrix:
         java: [ '11' ]
-        os: [ windows-latest, ubuntu-latest ]
+        os: [ windows-latest, ubuntu-latest, ubuntu-24.04-arm ]
```

Comment:
we definitely don't need this, the whole point was to make sure windows and non-windows are handled correctly.

Reply:
removed